xsl/xsl-ns-stylesheets/xhtml/chunk-common.xsl

```diff
@@ -1209,7 +1209,7 @@ xmlns:exsl="http://exslt.org/common" xmlns:cf="http://docbook.sourceforge.net/xm
     <xsl:if test="//d:legalnotice                   and not($generate.legalnotice.link = 0)                   and not($html.head.legalnotice.link.types = '')">
       <xsl:call-template name="make.legalnotice.head.links"/>
     </xsl:if>
-
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/docsearch.js/1/docsearch.min.css" />
```

Comment:
Yes! There are now two targets in the Makefile, webhelp (with DocSearch, for public doc) and webhelp-search (which includes the build-in doc, used previously, for embedded doc, it also remove all calls to DocSearch from the doc).